sdk/js/generated/api-definition.json

```diff
@@ -1543,26 +1543,6 @@
         "downlink_sent.path",
         "format",
         "headers",
-        "health_status",
```

Comment:
This diff shouldn't be there. You should be able to remove it via `mage jsSDK:definitions`.